src/Microsoft.DotNet.Wpf/ApiCompat/Baselines/PresentationFramework-ref-Net48.baseline.txt

```diff
@@ -1,15 +1,23 @@
 Compat issues with assembly PresentationFramework:
+MembersMustExist : Member 'Microsoft.Win32.FileDialog.CreateVistaDialog()' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'Microsoft.Win32.FileDialog.ProcessVistaFiles(MS.Internal.AppModel.IFileDialog)' does not exist in the implementation but it does exist in the contract.
```

Comment:
This is ApiCompat vs .NET 4.8 assemblies.  This is saying, "This exists in 4.8 but not in .NET Core".  That's why it's baselined because we are intentionally removing this from the ref surface area.

See: https://github.com/dotnet/wpf/blob/master/Documentation/api-compat.md